### ISSUE TITLE
No "Download selected" when you select mp3s in media section/text msgs in between

### ISSUE THREAD

**Issue by Aphotic** [telegramdesktop/tdesktop#26866]
### Steps to reproduce

found new bugs related to the fixes we got on #26761 & #26762 @23rd

no "download selected dialog" appears if you select files in the media section
![image](https://github.com/telegramdesktop/tdesktop/assets/30921064/3574bd3d-cfaa-4bae-9c10-11ed21308f85)

"no download selected dialog" appears if there are text msgs in between your selection, text msgs or anything that cannot be downloaded should just be ignored.

![image](https://github.com/telegramdesktop/tdesktop/assets/30921064/bfbb5343-e2db-4f75-a240-79bfba800bf3)
![image](https://github.com/telegramdesktop/tdesktop/assets/30921064/8b2a5c18-93d3-4dca-a01a-f79566aa0ec7)

### Expected behaviour

download selected should appear by any means if the media we selected are downloadable no matter what type of media they are. gifs/videos/ in case of music especially it is very useful to download playlists for example.

### Actual behaviour

no download selected appears.

### Operating system

Windows 10

### Version of Telegram Desktop

4.10 x64

### Installation source

Static binary from official website

**Comment by poliandro** [comment]
I'm on Windows, I have Telegram v, 4.10 x64. Inside media section, if I select multiple files, there is a menu called `Save As...`
Unfortunately it only downloads one file, even if multiple files are selected


![1](https://github.com/telegramdesktop/tdesktop/assets/9403993/115aa7cf-2b24-4efd-a068-e8a197b0fbdf)

**Comment by AFR98** [comment]
the reason you have "Save As" option is you already downloaded them.

i guess this is more of a feature request than a bug! @Aphotic

**Comment by poliandro** [comment]
Not true. This option appears regardless if the files were previously downloaded or not. The files not downloaded before have an arrow pointing down, the files already downloaded don't have the arrow. When messages are checked, the check sign covers the arrow.

**Comment by Aphotic** [comment]
this still happening:
![image](https://github.com/telegramdesktop/tdesktop/assets/30921064/c4278835-895f-44b4-9fe9-e57f5c96cd7a)
please do fix 
make the downloader ignore msgs/text or anything that cannot be downloaded
multiple different media types should still be allowed to be downloaded if its selected together
